cranelift/codegen/src/machinst/abi.rs

```diff
@@ -817,6 +819,7 @@ impl SigSet {
             ArgsAccumulator::new(&mut self.abi_args),
         )?;
         let rets_end = u32::try_from(self.abi_args.len()).unwrap();
+        let sized_stack_ret_space = u32::try_from(sized_stack_ret_space).unwrap();
```

Comment:
If you want, it would be cool to push the `u32`s back even further, so that `M::compute_arg_locs` returns a `u32` for the stack ret space and all the implementations for each backend use `u32`s and all that. Could be a follow up PR.

Reply:
Hah, @tnachen wanted to do that and I recommended going for the smaller patch first.